src/OrchardCore.Modules/OrchardCore.Contents/Endpoints/Api/CreateEndpoint.cs

```diff
@@ -46,6 +46,11 @@ public static IEndpointRouteBuilder AddCreateContentEndpoint(this IEndpointRoute
             return httpContext.ChallengeOrForbid("Api");
         }
 
+        if (model is null)
```

Comment:
@hishamco in what case will model be null here? I think the binding engine will create an instance on every request